contents/blog/posthog-restack-announcement.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2022-05-09
+title: Restack joins the PostHog Marketplace
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+categories: ["Product updates"]
+---
+
+Over the last few months, we’ve steadily grown the number of businesses offering services through [the PostHog Marketplace](/marketplace). We’ve added partnerships with teams such as [Altinity](/blog/posthog-altinity-announce), Birch, TenRec and, as of today, [Restack](/marketplace/restack). 
```

Comment:
```suggestion
Over the last few months, we’ve steadily grown the number of businesses offering services through [the PostHog Marketplace](/marketplace). We’ve added partnerships with teams such as [Altinity](/blog/posthog-altinity-announce), Birch, TenRec and, as of today, [Restack](/marketplace/restack). 
```

Isn't Tenrec still 'coming soon'?

Reply:
```suggestion
Over the last few months, we’ve steadily grown the number of businesses offering services through [the PostHog Marketplace](/marketplace). We’ve added partnerships with teams such as [Altinity](/blog/posthog-altinity-announce), Birch and, as of today, [Restack](/marketplace/restack). 
```

I mean, it's still listed, but good point. Have removed.